doc/whatsnew/2.12.rst

```diff
@@ -82,6 +86,8 @@ New checkers
 Extensions
 ==========
 
+* Pyreverse - add output in mermaid-js format and html which is an mermaid js diagram with html boilerplate
```

Comment:
This still needs to be moved to ``doc/whatsnew/2.13.rst`` I think, as Pierre set the milestone to 2.13.0 for this PR.
@Pierre-Sassoulas or do you plan to include it in an upcoming 2.12.x release?

Reply:
Sorry, it was an oversight. Thank you for your reviews.